{"name": "测试用例描述：错误的验证码登录失败", "status": "passed", "description": "测试描述：错误的验证码登录失败", "steps": [{"name": "输入正确账号", "status": "passed", "attachments": [{"name": "正确账号", "source": "50f2c0d6-eb3d-48ba-bd63-85e58fef7917-attachment.attach"}], "start": 1651819002813, "stop": 1651819002813}, {"name": "输入正确密码", "status": "passed", "attachments": [{"name": "正确密码", "source": "87617a9d-0bda-4859-afcc-10d1334dc8e2-attachment.attach"}], "start": 1651819002852, "stop": 1651819002852}, {"name": "输入错误验证码", "status": "passed", "attachments": [{"name": "验证码为空", "source": "dd26cd4b-0491-433d-8674-91f5309a0289-attachment.attach"}], "start": 1651819002902, "stop": 1651819002903}], "attachments": [{"name": "截图", "source": "444e6d7f-d867-499c-88ba-e375dffbb8ab-attachment.png", "type": "image/png"}], "start": 1651819002813, "stop": 1651819005476, "uuid": "9f6fcfd1-d07e-4131-87b0-b80613175501", "historyId": "99cc6b199ab7eac5979ecaecd144eda1", "testCaseId": "44097febf60fbfed56a361cecfc05a9e", "fullName": "test_two.Test#test_07", "labels": [{"name": "story", "value": "登录"}, {"name": "epic", "value": "xxx测试"}, {"name": "feature", "value": "登录模块"}, {"name": "severity", "value": "critical"}, {"name": "suite", "value": "test_two"}, {"name": "subSuite", "value": "Test"}, {"name": "host", "value": "MS-SLSXLICKSUSF"}, {"name": "thread", "value": "9800-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_two"}]}